target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -49,6 +49,9 @@ ramips_board_detect() {
 	*"ALL5003")
 		name="all5003"
 		;;
+        *"APAC1200")
```

Comment:
Drop it to use the generic board detection. The userspace board name will be `v7,apac1200`.